src/sets.jl

```diff
@@ -184,6 +184,28 @@ constant(s::EqualTo) = s.value
 constant(s::GreaterThan) = s.lower
 constant(s::LessThan) = s.upper
 
+"""
+    NormInfinityCone(dimension)
+
+The epigraph of the ``\\ell_\\infty``-norm function ``\\{ (t,x) \\in \\mathbb{R}^{dimension} : t \\ge || x ||_∞ = \\max_i |x_i| \\}`` of dimension `dimension`.
```

Comment:
I meant the infinity symbol in the subscript of the norm. By the way, the norm should be `\|` or `\lVert`/`\rVert` instead of `||`